assets/helpers/page/page.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { createStore, combineReducers } from 'redux';
+import 'ophan';
+
+import * as ga from 'helpers/tracking/ga';
+import * as abTest from 'helpers/abtest';
+import * as logger from 'helpers/logger';
+import { getCampaign } from 'helpers/tracking/guTracking';
+import { getQueryParameter } from 'helpers/url';
+import { detect } from 'helpers/internationalisation/country';
+
+import type { Campaign } from 'helpers/tracking/guTracking';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { Participations } from 'helpers/abtest';
+
+import type { Action } from './pageActions';
+
+
+// ----- Types ----- //
+
+type CommonState = {
+  intCmp: ?string,
+  campaign: ?Campaign,
+  refpvid: ?string,
+  country: IsoCountry,
+  abParticipations: Participations,
+};
+
+
+// ----- Functions ----- //
+
+function pageInitialisation() {
```

Comment:
I don't think it is intuitive that `pageInitialisation()` returns `participations`.

It might make sense to rename this `analyticsInitialisation` and pass in `participations`

Reply:
we could also include the return type to increase readability.

Reply:
ac2a0a1

Reply:
👍